### DISCUSSION TITLE
How to hide feedback widget tooltip?

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [squidfunk/mkdocs-material#4287] [Q&A] [answered]
When you hover over the "Was this page helpful?" icons a tooltip appears. When you click an icon, the tooltip doesn't disappear and it covers the "thank you" note. Is there a way to make the tooltip disappear after the icon is clicked? 

<img width="754" alt="Screen Shot 2022-08-27 at 9 15 31 pm" src="https://user-images.githubusercontent.com/9379524/187028304-084fc55c-1a7d-42cc-8e22-85b38998ddd3.png">

**Comment by squidfunk** [comment, marked as answer]
This doesn't happen on my machine, tested in Chrome. When I click the button, the tooltip disappears. Is there an error in the console after you've clicked the icon?

**Reply by Stephen-Gates** [reply]
It works for me in Chrome and Brave but not in Safari 15.6. I cannot see any errors in the console.

**Reply by squidfunk** [reply]
Confirmed. Please open an issue, that's definitely not intended behavior.

**Reply by Stephen-Gates** [reply]
Done https://github.com/squidfunk/mkdocs-material/issues/4288
